### DISCUSSION TITLE
Funding the future of this valuable project.

### DISCUSSION THREAD

**Discussion by ajnienaber** [codeproject/CodeProject.AI-Server#187] [General]
I'm shocked that it's so hard to find info anywhere about the future of CodeProject in the community.
Is there a method that we the users can help fund this project in the here and now?

**Comment by ChrisMaunder** [comment]
I will try and keep everyone up to date on where things sit but for now I don't have a lot of insight. However, I'm continuing to ensure we keep the project alive, but it is purely on a volunteer basis, and I am unable to arrange or speak for any plans regarding sponsorships of the project. For now just assume business as usual. Just slightly slower than usual ;)

**Comment by mag1024** [comment]
Your work is greatly appreciated!
Have you considered approaching Blue Iris about a possible sponsorship arrangement?
Seems like the AI server is basically a requirement for BI these days....
